manifests/latest/vsphere-7.0u1/vanilla/deploy/create-validation-webhook.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -e
+
+if [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
+  cat <<EOF
+Usage: Patch validatingwebhook.yaml with CA_BUNDLE retrieved from Kubernetes API server
+and create ValidatingWebhookConfiguration and scvalidation-webhook-svc service using patched yaml file
+EOF
+  exit 1
+fi
+
+CA_BUNDLE=$(kubectl get configmap -n kube-system extension-apiserver-authentication -o=jsonpath='{.data.client-ca-file}' | base64 | tr -d '\n')
+
+# clean-up previously created service and validatingwebhookconfiguration. Ignore errors if not present.
+kubectl delete validatingwebhookconfiguration.admissionregistration.k8s.io scvalidation.csi.vsphere.vmware.com 2>/dev/null || true
```

Comment:
The resource name for validatingwebhookconfiguration & the service should match the names in validatingwebhook.yaml

Reply:
done.